crates/configuration/src/main.rs

```diff
@@ -0,0 +1,12 @@
+use configuration::{ConfigError, NetworkConfigBuilder};
+
+pub fn main() -> Result<(), ConfigError> {
+    let network_config = NetworkConfigBuilder::new()
+        .with_global_settings(|g| g)
+        .with_relaychain(|r| r)
```

Comment:
I think we can add another method here, `with_default_relaychain` that just change the `type state`, allowing users to use the default `rococo-local` (w/o any customization).